.travis.yml

```diff
@@ -23,14 +30,17 @@ install:
   - popd
 
 before_script:
+  - sudo sed -i -e '/local.*peer/s/postgres/all/' -e 's/peer\|md5/trust/g' /etc/postgresql/11/main/pg_hba.conf
+  - sudo systemctl restart postgresql@11-main
```

Comment:
- [ ] Why need `sed` pg_hba.conf ? I couldn't found reason in #863, #868 .

Reply:
sorry. I didn't leave a reason for the correction.  
Because of Peer authentication, it was not possible to connect to postgresql locally, so it needed to be configured.
I have determined that the event is the same as that reported in the following issue.
https://github.com/travis-ci/travis-ci/issues/9624#issuecomment-389537036

and I added the reason why I set it to https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/issues/863#issuecomment-745091742 .